src/components/core/a11y/focus.ts

```diff
@@ -17,23 +17,23 @@ export const IS_FOCUSABLE_QUERY = [
 export function getFocusableElements(
   elements: HTMLElement[],
   properties?: {
-    filterFunc?: (el: HTMLElement) => boolean;
+    filter?: (el: HTMLElement) => boolean;
     findFirstFocusable?: boolean;
     includeInvisibleElements?: boolean;
   },
 ): HTMLElement[] {
   const focusableEls = new Set<HTMLElement>();
 
-  function getFocusables(elements: HTMLElement[], filterFunc?: (el: HTMLElement) => boolean): void {
+  function getFocusables(elements: HTMLElement[], filter?: (el: HTMLElement) => boolean): void {
     for (const el of elements) {
-      if (filterFunc?.(el)) {
+      if (filter && !filter(el)) {
         continue;
       }
```

Comment:
Yes I intentionally inverted the logic because normally using a filter you define what you want to keep and not what you like to ignore. So now it should be lesse confusing to define a filter :-)